\documentclass[rmp, reprint, superscriptaddress, floatfix,amsmath]{revtex4-1}
\usepackage{longtable}
\usepackage{color}
\usepackage{graphicx}  % needed for figures
\usepackage[utf8x]{inputenc}
\usepackage{hyperref}
\usepackage{longtable}
\usepackage{xcolor}
\usepackage{soul}
\usepackage{multirow}
\usepackage{longtable}


\definecolor{drab}{rgb}{0.59, 0.44, 0.09}
\newcommand{\Richard}[1]{{\color{drab}Richard: #1}}
\definecolor{celestialblue}{rgb}{0.29, 0.59, 0.82}
\newcommand{\Robert}[1]{{\color{celestialblue}Robert: #1}}
\definecolor{purple}{rgb}{0.459,0.109,0.538}
\definecolor{deepsaffron}{rgb}{1.0, 0.6, 0.2}
\newcommand{\Jan}[1]{{\color{deepsaffron}Jan: #1}}
\newcommand{\Emma}[1]{{\color{purple}Emma: #1}}
\definecolor{green}{rgb}{0.15, 0.6, 0.15}
\newcommand{\Valentin}[1]{{\color{green}Valentin: #1}}


\begin{document}

\title{Impact of seasonal forcing on a potential SARS-CoV-2 pandemic}

\author{Richard A.~Neher}
\address{Biozentrum, University of Basel, Basel, Switzerland \looseness=-5}
\address{Swiss Institute of Bioinformatics, Basel, Switzerland \looseness=-5}
\email{Correspondence to: Richard Neher, Biozentrum, Klingelbergstrasse 70, 4056, Basel, Switzerland. \bf{richard.neher@unibas.ch}}
\author{Robert Dyrdak}
\address{Department of Clinical Microbiology, Karolinska University Hospital, Stockholm, Sweden \looseness=-7}
\address{Department of Microbiology, Tumor and Cell Biology, Karolinska Institute, Stockholm, Sweden \looseness=-7}
\author{Valentin Druelle}
\address{Biozentrum, University of Basel, Basel, Switzerland \looseness=-5}
\address{Swiss Institute of Bioinformatics, Basel, Switzerland \looseness=-5}
\author{Emma B.~Hodcroft}
\address{Biozentrum, University of Basel, Basel, Switzerland \looseness=-5}
\address{Swiss Institute of Bioinformatics, Basel, Switzerland \looseness=-5}
\author{Jan Albert}
\address{Department of Clinical Microbiology, Karolinska University Hospital, Stockholm, Sweden \looseness=-7}
\address{Department of Microbiology, Tumor and Cell Biology, Karolinska Institute, Stockholm, Sweden \looseness=-7}

\date{\today}

\begin{abstract}
A novel coronavirus (SARS-CoV-2) first detected in Wuhan, China, has spread rapidly since December 2019, causing over 43,000 confirmed infections and more than 1,000 fatalities (as of Feb 12, 2020). Imported cases and small transmission clusters have been reported globally.
Early data suggests the virus transmit readily and a pandemic cannot be ruled out.

Based data from routine diagnostic showing a strong and consistent seasonal variation of the common coronaviruses (229E, HKU1, NL63, OC43), we explore the effect of seasonal variation in transmissibility on a potential SARS-CoV-2 pandemic. 
A model allowing for many subpopulations of different size with variable parameters of SARS-CoV-2 spread shows how a pandemic could unfold in 2020-2022.
Simulations of different scenarios show that plausible parameter result in a peak in temperate regions of the Northern Hemisphere in winter 2020/2021. 
A smaller range of parameters suggest a peak in the first half of 2020 or two peaks of similar magnitude. 
Variation in transmission and migration rates can result in substantial variation in prevalence between regions.

While the uncertainty in parameters is large, the scenarios we explore show that transient reductions in the incidence rate might be due to a combination of seasonal variation and infection control efforts but do not necessarily mean the epidemic is contained. 
Seasonal forcing on SARS-CoV-2 should thus be taken into account in the further monitoring of the global transmission. 
The likely aggregated effect of seasonal variation, infection control measures, and transmission rate variation is a prolonged pandemic wave with lower prevalence at any given time, thereby providing a window of opportunity for better preparation of health care systems.
%world-wide by scaling up capacityfor care and diagnostics reducing the pressure on health care systems and supply chains. 

\end{abstract}
\maketitle


\section{Introduction}

On Jan 30, 2020, the World Health Organisation (WHO) declared the spread of a new coronavirus, SARS-CoV-2 (provisionally named 2019-nCoV \citep{ICTV_SARS-CoV-2}), as a public health emergency of international concern \citep{WHO2020}.
%, and on Feb 11 that it ``holds a very grave threat for the rest of the world" \Robert{can't find this on official WHO web page, might we reference to e.g. Reuter or AFP?}. 
%\Jan{We can skip the previous sentence}
The virus was first identified in patients with pneumonia in the city of Wuhan in the Hubei province, China, in December 2019 \citep{Liangjun2020rna}. % (and the new virus was identified by metagenomic next generation sequencing in two patients with pneumonia) . 
%The new coronavirus was shown to be closely related (89.1 \% nucleotide similarity) to a group of SARS-like coronaviruses (genus Betacoronavirus, subgenus Sarbecovirus) previously sampled from bats in China. \Jan{ref to Wu Nature}
The clinical presentation of the illness caused by SARS-CoV-2, called COVID-19,
%\Jan{Capitals see https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200211-sitrep-22-ncov.pdf}
appears to range from mild or asymptomatic to severe and fatal respiratory illness \citep{WHO2020situation_report}, but the exact spectrum of disease presentation is still unclear. %\Jan{We could skip the following paragraph if needed}
%As of Feb ?, XXXXX confirmed cases have been reported, including YYY fatal cases (to be updated closer to submission). Most of the fatal cases have been reported from Wuhan and the Hubei province in China. At present, mortality appears to be lower in cases identified outside of China. Imported cases of nCoV were reported from X countries in Europe \Robert{as of 7 Feb: nine} (ref: ECDC). \Robert{also mention no of countries in Asia and spread to N America and Oceania?}  \Jan{Yes mention these}
The potential for global spread, i.e.~a pandemic, of SARS-CoV-2 is currently not known, but the virus has spread at an alarming rate in Wuhan, the epicenter of the outbreak. Furthermore, the virus has spread to almost all provinces of China \citep{ECDC_situation_update} and small clusters of local spread have been reported from Germany \citep{rothe2020transmission}, Singapore \citep{MOH_Singapore}, and the UK \citep{WHO_situation_report}.
%\Robert{Emma, could we reference your compilation of the UK cases? maybe push it on bioRxiv, or if you can put it on Basel univ homepage?}... \Emma{Definitely open the idea... Can you put images on bioRxiv? I can put on my own homepage if nothing else, but that would just be a web-link.}
%On Jan 28, 2020, a cluster including two locally (within a company) acquired infections, was reported from Germany\citep{rothe2020transmission}. On Feb 7, the Singapore Ministry of Health reported three cases with no known links to previous cases or travel history to China, suggesting a local transmission (https://www.moh.gov.sg/news-highlights/details/three-more-confirmed-cases-of-novel-coronavirus-infection-in-singapore). 

%Analysis of 27 whole genome sequences showed positive selective pressure in the spike protein S responsible for virus entry into the cell, and of nucleocapsid N protein involved in virus transcription and virion assembly \citep{benvenuto20202019}.
%\Jan{Not sure that this is needed}

%Modelling of R0 \citep{tuite2020reporting}

The basic reproduction number ($R_0$), which describes the average number of new infections per infected case, has been estimated to be around 2.2 (90\% high density interval: 1.4–3.8) or 2.7 for SARS-CoV-2 \citep{Riou2020pattern,wu_nowcasting_2020}, while others report higher estimates \citep{yang_epidemiological_2020}. Importantly, $R_0$ is not a biological constant for a pathogen, but is affected by factors such as environmental conditions and the behaviour infected individuals. One such environmental factor is climate. Many viral infections show clear seasonal variation; the most well-known example being seasonal influenza which peaks every winter in the temperate zone of the Northern Hemisphere \citep{petrova_evolution_2018}. 
A similar pattern is seen for the four seasonal human coronaviruses: HKU1, NL63, OC43 and 229E (hereafter collectively referred to as ``seasonal CoVs'') \citep{killerby2018human} %\Robert{USA, peak in December to March}
\citep{goes2019typical} %\Robert{Brazil, says "Winter" in the figure, in text winter is defined as June 21st - Sept 21st (dry, cold)} 
\citep{galanti2019longitudinal} %\Robert{New York, peak in winter, no cases during summer}
\citep{friedman2018human} %\Robert{Israel, detection Oct to March}
\citep{al2016diversity} %\Robert{Malaysia, peak in June to October}
\citep{huang2017epidemiology} %\Robert{Taiwan, NL63, Jan-Feb 2012} 
%\Robert{this is more of a result I think: It is reasonable to assume that 2019-nCoV will also show seasonal variation with winter peaks on the Northern Hemisphere if the current outbreak will lead to endemicity. If the efficiency of transmission of 2019-nCoV is influenced by outdoor temperature this may also affect the development of a possible future 2019-nCoV pandemic.} 
%The risk of importation to Europe was estimated to be high, in particular if seeing area included Shanghai and Beijing \citep{pullano2020novel}. Given the current situation (to do: check current status of cases in Beijing and Shanghai)
%There are four human corona virus known to cause rather mild respiratory illness (229E, OC43, NL63, HKU1), which are globally ubiquitous present, and two corona viruses causing more severe and fatal disease (SARS-CoV, and MERS-CoV). The nCoV belong to betacoronavirus, and is most closely related to SARS-like bat viruses (ref). 
%Also, circulation of MERS-CoV in Middle East is found to have coincided with circulation of influenza \citep{he2015differences} \Robert{peaks in May two seasons}
%\Jan{Suggest we save this for discussion}
%In the case of the pandemic influenza (H1N1) virus in 2009 in Sweden, cases with infection contracted abroad appeared in July and caused a wave in which peaked in week 29. In this first wave, local spread was limited and accounted for only about 20\% of the cases. A second wave, coinciding with return from vacations, peaked in week 36 and declined until week 40 when an extensive spread commenced \citep{SMI2011}.             

Previous influenza pandemics have swept the world in multiple waves often but not always coinciding with winter month in temperate climates \citep{taubenberger_1918_2019,viboud_multinational_2005,viboud_global_2016}. 
\Richard{Add more on influenza}. \Emma{Emma will work more on this section.}
The 1968-1970 global influenza pandemic was sparked by a new influenza A/H3N2 virus, with a divergent hemagglutinin protein, which replaced the A/H2N2 virus circulating for 10 years previously \citep{viboud_multinational_2005}. The virus spread rapidly, but the viral dynamics and morality was not equal across countries \citep{viboud_multinational_2005}. While many experienced two more-severe flu seasons in the winter seasons of 1969 and 1970, the US had higher mortality in the first season, while European countries, Japan, and Australia, had higher mortality in the second, and Canada had roughly equal mortality in both.

The 2009 pandemic H1N1 virus originated in March 2009 in Mexico and spread around the globe within weeks.

\Robert{e.g. something like ´´In Sweden 2009, the introduction of pandemic influenza A H1N1 in the late fall resulted in two transient minor peaks, before the major peak and epidemic in autumn. \citep{SMI2011}"?}
\Emma{Add something about H1N1 transition to seasonality? And its double-peak? (Like graphs robert sent.)}

Here we use data on seasonal variation of seasonal CoVs and model the impact of this variation on the possible future spread of SARS-CoV-2 in the temperate zone of the Northern Hemisphere. 
We also explore different scenarios of SARS-CoV-2 spread in temperate and tropical regions and show how variation in epidemiological parameters affects the a potential pandemic and a possible transition to an endemic state. 

\section{Seasonal coronavirus prevalence} 

Data on seasonal variation of HKU1, NL63, OC43 and 229E diagnoses in respiratory samples was obtained from the routine molecular diagnostics at the Karolinska University Hospital, Stockholm, Sweden. The laboratory provides diagnostic services to six of seven major hospitals and approximately half of outpatient care in the Stockholm county (2.2 million inhabitants). We extracted pseudonymized data on all analyses for the four viruses between Jan 1, 2010 and Dec 31, 2019. The data set included a total of 52,158 patient samples with 190,257 diagnostic tests \Robert{double check final numbers}, of which 2,084 were positive for either of the coronaviruses (229E = 319; NL63 = 499; OC43 = 604; HKU1 = 355; OC43/HKU1 = 307). Metadata included information about date of sampling and age of patient. 
In the period Jan 1, 2010 to Nov 5, 2017, the coronavirus diagnostic was done using in-house assays \citep{tiveljung2009development}. From Nov 6, 2017 to Dec 31, 2019, samples were analysed using the commercial kit Allplex Respiratory Panels (Seegene Inc., Seoul (South Korea)). This commercial kit does not distinguish between HKU1 and OC43 and for this reason positive tests for these two viruses were combined for the entire study period. 

The fraction of tests that were positive for the four seasonal CoVs showed a strong and consistent seasonal variation, see Fig.~\ref{fig:seasonal_CoV}. 
From December to April approximately 2\%  of tests \Jan{Are we looking at samples or tests?
} were positive, while less than 0.05\% of tests were positive between July to September (Fig.~\ref{fig:seasonal_CoV}B).
The strength of variation of the transmission rate through the year could be of high relevance to the spread of SARS-CoV-2 in 2020 and following years. 

\begin{figure*}
    \centering
    \includegraphics[width=\columnwidth]{figures/pos_frac_vs_time.pdf}
    \includegraphics[width=\columnwidth]{figures/pos_frac_by_month.pdf}
    \caption{Seasonal variation in fraction of positive CoV test in Stockholm, Sweden.}
    \label{fig:seasonal_CoV}
\end{figure*}

\section{Basic model}
We consider simple SIR models of the form
\begin{equation}
\begin{split}
\frac{d}{dt} S & = b(t)(1-S) - \beta S I \\
\frac{d}{dt} I & = -(\nu+b+e) I + \beta S I + i\\
R & = 1-S-I
\end{split}
\label{eq:SIR}
\end{equation}
where $\beta$ is the rate at which an infected individual infects a susceptible one, $\nu$ is the inverse serial interval, $b$ is the population turn-over rate, $e$ is the emigration rate, and $i$ is the influx of infected individuals.
The population turnover rate is immaterial for a pandemic scenario, but important for our analysis of seasons CoV and should be interpreted as the sum of the birth rate and the rate at which previously immune individuals become susceptible due to immune waning and escape.
We review general properties of such model in the supplement. 
Following previous work, we parameterize transmissibility as 
\begin{equation}
\beta(t) = \beta_0\left(1+\epsilon\cos(2\pi (t-\theta))\right)
\label{eq:transmissibility}
\end{equation}
where $\beta_0$ is the annual average, $\epsilon$ is the amplitude of seasonal variation, and $\theta$ is the time of peak transmissibility \citep{dushoff_dynamical_2004,chen_regular_2017}.
For simulations of the pandemic, we will add an additional term to $\beta(t)$ that accounts for infection control measures in heavily affected areas, see Supplementary Materials.

\begin{figure}
    \centering
    \includegraphics[width=\columnwidth]{figures/fit_2.5.pdf}
    \caption{{\bf Compatibility of SIR model trajectory with observations.} The heatmap shows the inverse mean squared deviation between the model trajectories and the seasonal variation in  seasonal CoV prevalence. Model and observations are compatible in a region of parameter values corresponding to low migration/weak seasonality and second region at high migration/strong seasonality. }
    \label{fig:parameter_seasonal}
\end{figure}

\section{Model parameterization using seasonal CoV observations}
Seasonal CoVs are endemic throughout the world and we therefore expect that viruses are imported throughout the year. 
We model this import as a constant influx $i$. 
Furthermore, we assume that humans suffer from an seasonal CoV infection on average every 10 years ($b=0.1/y$), $R_0=2.5$ and a recovery rate (time between infections) of 10 days.
\Jan{Do we know anything about long-standing immunity? Is the assumption of 10 years between infections realistic? Is this assumption compatible with Robert´s extracted data?}
\Richard{We don't know. 10\% attack rate is a typical guess for flu, but we can explore other parameters.}
\Robert{doesn't seem to be much in the literature on the long-lasting immunity for CoVs...}
\Jan{The over-representation of children <5y certainly argues for some type of immunity protecting older persons. Let's stick with Richard's assumptions}

With these assumptions, we can solve the model and compare the resulting trajectories to the seasonal variation in prevalence of seasonal CoVs, see Fig.~\ref{fig:parameter_seasonal}.
Simulations of the model are compatible with observations in two separate regions of parameter space: If the Northern Europe was very isolated with less than 1 in 1,000 individuals returning with a seasonal CoV infection from abroad, weak seasonality around $\epsilon=0.15$ would be sufficient to generate strong variation through the year compatible with observations (Fig.~\ref{fig:parameter_seasonal}, bottom-left ridge).
In this regime, prevalence is oscillating intrinsically with a period that is commensurate with annual seasonal oscillations giving rise to a resonance phenomenon with annual or biennial patterns even for weak seasonal forcing \citep{dushoff_dynamical_2004,chen_regular_2017}.

\Emma{Emma re-read this and ensure ok for non-specialists.}
If the rate of import of seasonal CoV infections is higher, imports dampen the resonance and much stronger seasonality with values between $\epsilon=0.3$ and $0.7$ are required to fit the observations (Fig.~\ref{fig:parameter_seasonal}, top-right-and-center ridge).
In this regime seasonal variation in transmissibility modulates the size of micro-outbreaks triggered by imported cases in a mostly immune population. 

These two scenarios differ slightly in the time of year at which peak transmissibility $\theta$ occurs: 
When transmission is mostly local and seasonality is amplified by resonance, $\theta$ needs to be around October--November to fit the data. 
In the second scenario with high connectivity, $\theta$ needs to be December--January. 
Given that most countries are highly connected, we focus here on exploring the high-import and seasonal forcing scenario.
This scenario with maximal $\beta$ in mid-winter is also more compatible with climate variation around the year.

%\textit{Given that we observe a peak in December--January our simulation suggests relatively high connectivity, \textit{i.e.} stronger seasonality.}  \Jan{Last sentence OK?}
%\Emma{No, think last sentence takes too far.. But we need a sentence about why we focus on high-season/high-migration.}


\begin{figure*}
    \centering
    \includegraphics[width=\columnwidth]{figures/scenarios.pdf}
    \includegraphics[width=\columnwidth]{figures/peak_ratio_0.5.pdf}
\caption{{\bf Model predictions for SARS-CoV-2 case numbers in temperate zones for a pandemic scenario}. The left panel shows example trajectories assuming SARS-CoV-2 transmissibility peaks in November, January, or March. 
These outbreaks in Northern Europe (`NE') are assumed to be seeded by the outbreak in Hubei (model trajectory shown as a dashed line).
Within the model, these cases are exported at rate of 0.01/year to temperate Northern Europe with an average $\langle R_0\rangle = 1.8$ and seasonal forcing of $\epsilon=0.5$. Corresponding graphs for different values of $\langle R_0\rangle$ and the migration rate are shown in Fig.~\ref{fig:scenarios_supp}.
The right panel shows the ratio of first and second peak for a range of different combinations of $R_0$ and $\theta$. 
The yellow area corresponds to parameter combinations with essentially only an early peak similar to the yellow line on the left. 
The blue area shows parameter combinations for which a peak in late 2020 dominates, as with the blue line on the left, while the central pink/orange band shows the combinations giving rise to two comparable peaks. 
These simulations are for $\epsilon=0.5$. Similar results were obtained for $\epsilon=0.3$ and 0.7, see Fig.~\ref{fig:peak_ratio_supp}.}
    \label{fig:nCov_predictions}
\end{figure*}


\section{Scenarios for SARS-CoV-2 pandemics in 2020 and 2021}
The analysis of seasonal CoV prevalence patterns allowed us to constrain parameter ranges and explore different scenarios of SARS-CoV-2 spread around the globe, in particular in temperate climates like Northern Europe. 
In the following we will explore scenarios where temperate regions have a seasonal forcing between $\epsilon=0.3$ and $0.7$ and migration rates of $0.01$/year. 
Early estimates suggest and incubation time of about 5 days \citep{backer_incubation_2020} and an average serial interval of 7-8 days \citep{wu_nowcasting_2020}, suggesting rapid transmission once symptoms set in. 
Our model does not account for the incubation time and therefore generates too many early transmissions. 
We therefore use a slightly lower recovery rate of $\nu = 0.1/day$. 
To match the $R_0$ estimates for the early outbreak with our parameterization of transmissibility in Eq.~\ref{eq:transmissibility} we need to account for the fact that December/January are winter months in Hubei and peak transmissibility in Hubei likely corresponds to $\theta=0$. \Jan{Isn't this an indication that the spread in Hubei and other parts of Northern China may be expected to slow down in the spring due to seasonality? Maybe mention in discussion.}
An $R_0\approx 2.5$ in winter in Hubei and a seasonal forcing of $\epsilon=0.4$ implies and annual average $\langle R_0\rangle = \beta_0/\nu=1.8$.
This reasoning leads to our parameter choice $\beta_0=65$/year, $\nu=36$/year, $theta=0$. 
We assume the outbreak started at $t=2019.8$ in Hubei with one infected individual.
Transmissibility is reduced by 50\% once prevalence reached 1\% (third order Hill-function, see supplement).
Introductions to a location like Northern Europe with $\epsilon=0.5$ are assumed to happen at a rate of 0.01 per year for each infected individual elsewhere.
The simulation of the SIR model in different regions is deterministic, but migration is implemented stochastically by Poisson resampling of the average number of migrating individuals. 

Fig.~\ref{fig:nCov_predictions} shows simulated trajectories of SARS-CoV-2 prevalence in the temperate Northern Hemisphere assuming the outbreak started in Hubei early December 2019.
Depending whether the peak transmissibility of SARS-CoV-2 in temperate zone is in November, January, or March, the simulation predicts a main peak in the first half of 2020, a main peak in winter 2020/2021, or two similarly sized peaks.

To explore possible scenarios more systematically, we ran such simulations for a range of values for $R_0$ and peak transmissibility $\theta$ and recorded whether we observe and early peak, a late peak, or a two peaks. 
The right panel of Fig.~\ref{fig:nCov_predictions} shows the ratio of these height of these peaks for different values.
Rapid growth (high $R_0$) and late transmission peaks result in a large peak in the first half 2020, while lower $R_0$ and transmission peaks in early winter favor a large secondary peak. 
These two scenarios are separated by a band of parameter values that give rise to two pandemic waves in the winters of 2020 and 2021 in the Northern Hemisphere.

The uncertainty in parameter values and the potential impact of infection control measures imply that all scenarios are plausible and should be considered when developing pandemic prevention and containment strategies.


\section{Global projections}
Outbreaks grow exponentially within well-mixed communities, and at a certain rate the virus will be carried to other regions and potentially seed new outbreaks. 
Such export to new locations is initially unlikely, but becomes next to certain once the outbreak size exceeds the inverse probability that a given individual migrates while infected.

Every location has a different socio-economic profile such that the growth rate of the epidemic ($R_0$) might differ. 
The superposition of many such sub-populations with a range of $R_0$ values and seasonal variation in transmission will result in dynamics that are qualitatively different from a single population SIR model. 
In particular, such variation can allow the virus to persist for years before possibly becoming endemic. 

Fig.~\ref{fig:global_3_panel} shows the result of such a simulation of 1,000 populations.
Populations were divided between northern temperate (50\%), southern temperate (10\%), and tropical (40\%), with peak transmissibility in December/January, June/July, or randomly chosen, respectively. 
We assume a uniformly distributed seasonal forcing $\epsilon$ between 0.25 and 0.75 for temperate regions and between 0 and 0.2 for tropical regions. 
Each population was assigned a log-normally distributed population size, $\langle R_0\rangle$ was drawn from a normal distribution with mean $1.8$ and standard deviation $0.5$, and migration rate between these regions and the global pool was drawn from a log-normal distribution with mean $0.01$ per year. 

The variation in $R_0$ results in a super-position of fast and slow epidemics seeded at different times.
The initial phase is dominated by fast epidemics driving rapid dispersal, in particular in the tropics, while slow epidemics dominate later in 2020 and 2021.
With the parameter setting used in Fig.~\ref{fig:global_3_panel}, the Northern temperate regions see most circulation in winter 2020/2021. 
In accordance with Fig.~\ref{fig:nCov_predictions}, this peak shifts more towards early 2020 for higher $R_0$, see Fig.~\ref{fig:global_3_panel_supp}.

\begin{figure*}[tb]
    \centering
    \includegraphics[width=\textwidth]{figures/global_3_panel_1.8.pdf}
    \caption{{\bf Exended circulation through overlapping epidemics in variable subpopulations.} These simulations of a pandemic scenario assume 1,000 sub-populations with an average $\langle R_0=1.8\rangle$ and standard deviation $0.5$, 40\% of which have weak seasonal variation $\epsilon \in [0,0.2]$ (tropical) and the remainder strong variation with $\epsilon \in [0.25,0.75]$. The super-position of many variable epidemics can result in a global prevalence that decays only slowly through 2020 and 2021. 
    Lighter lines have lower $R_0$, darker lines have higher $R_0$. 
    The actual case counts reported for Hubei are added to the first panel and multiplied by three to account for possible under-reporting of mild cases.
    Analogous figures for different parameter values are shown in Fig.~\ref{fig:global_3_panel_supp}.}
    \label{fig:global_3_panel}
\end{figure*}

In the medium term, SARS-CoV-2 could become a seasonal CoV with characteristic winter outbreaks as shown in Fig.~\ref{fig:seasonal_CoV}.
Such a scenario is demonstrated in Fig.~\ref{fig:endemic} where a simulation similar to the one shown in Fig.~\ref{fig:global_3_panel} is run for 12 years, assuming that after infection an individual become susceptible to SARS-CoV-2 again at a rate of 0.1 per year. 
After a pronounced low in 2020-2024, prevalence recovers and settles into a seasonal pattern, similar to that of the four existing seasonal CoVs. 

\begin{figure}
    \centering
:    \includegraphics[width=\columnwidth]{figures/global_endemic_1.8.pdf}
    \caption{{\bf Transition to an endemic seasonal virus.} 
    If previously infected individuals can be reinfected after some time, as for example by seasonal influenza virus, SARS-CoV-2 could develop into a seasonal CoV that returns every winter. This would typically happen at much lower prevalence than peak pandemic levels. 
    This graph assumes reinfection on average every 10 years.}
    \label{fig:endemic}
\end{figure}

\section{Discussion}

We report on the possible influence on seasonal variation on the spread of SARS-CoV-2 in the Northern Hemisphere, should the current outbreak progress to a pandemic. We find that seasonal variation in transmissibility has the potential to modulate the spread of SARS-CoV-2 with a wide range of possible outcomes that need to be taken into account when interpreting case counts and projecting the outbreak dynamics. 
The onset of spring and summer could, for example, give the impression that SARS-CoV-2 has been successfully contained, %the fight against SARS-CoV-2 is won, % Emma - I just wasn't sure about this causl wording - but change back if you want
%Jan - Better like this
only for the virus to return \Robert{upsurge?} later in 2020-2021 winter season. 
Whether a pandemic in the temperate regions of the Northern Hemisphere would peak early in 2020, late in 2020, or show multiple waves as H1N1pdm did in 2009, depends on the timing of peak transmissibility and the rate of spread ($R_0$). \Jan{Isn't the serial interval also important, not just R0?}

This study is meant as an exploration of how such a pandemic could unfold, not as a prediction of any particular scenario.
The results we present are depend critically on the assumptions
i) that the outbreak will develop into a pandemic,
ii) that the transmissibility of SARS-CoV-2 shows seasonal variability of sufficient strength (range $\epsilon=0.3$ to $0.7$), and  
iii) that parameters like $R_0$ estimated from the early phase of the outbreak are comparable in other populations. 

These assumptions are not implausible but not certain: cases of SARS-CoV-2 has been detected in 26 countries/territories \citep{ECDC_situation_update}, and mild or asymptomatic cases make detection and thus prevention of spread by isolation challenging, \textit{e.g.} airport screening as a preventive measure is unlikely to prevent spread and local seeding \citep{quilty2020effectiveness}. 
Person-to-person transmission of the virus has been documented in several countries other than China. 
It is likely that not all exported cases have been detected, and some may have seeded outbreaks outside of China that have yet to be detected. It is estimated that if SARS-CoV-2 continues to spread in China, and particularly if the areas of Beijing and Shanghai are affected, the risk of importation of cases to Europe increases \citep{pullano2020novel}.

The seasonal CoVs show a strong and consistent seasonal variation, and modeling suggests that this requires strong variation in transmissibility throughout the year.
It should be noted, however, that SARS-CoV-2 does seem to transmit in tropical climates like Singapore, and so winter is not a necessary condition of SARS-CoV-2 spread. 
Precise values for the underlying model parameters and the effect of infection control measures are currently unavailable. For this reason we explored a range of parameter values to assess the robustness of the results to model assumptions.


The simulations presented here are scenarios that emerge from simplified abstract models, but they nevertheless demonstrate that a wide variety of outcomes are compatible within the limits of the current knowledge about the outbreak. 
The implications of our work are that: 1) reductions in the prevalence need not be attributable to successful interventions, but could be due to seasonal variation in transmissibility, 2) sub-population dynamics can differ greatly, meaning that case count trajectories in one country should be used cautiously to inform projections in a second country, even in the same climate zone; further, in the medium term SARS-CoV-2 could develop into an endemic seasonal CoV after several years similar to the transition of the 2009 A/H1N1 pandemic influenza virus into a seasonal flu, and 3) seasonal variation might slow down a pandemic and thereby provide a window of opportunity for better preparation of health care systems world-wide by scaling up capacity for care and diagnostics, and potentially through rapid development of antivirals and vaccines.

The overall impact of a potential SARS-CoV-2 pandemic depends critically on the case fatality rate (CRF), which we have not modelled here. 
At present, uncertainty in the CRF is high due to likely over-representation of severe cases in the statistics and a delay between recovery/death and diagnosis \citep{battegay_2019-novel_2020}. 
But seasonal variation in transmissibility of SARS-CoV-2 and underlying differences in migration, introduction times, and attack rate should thus be taken into account when monitoring and projecting global transmission, planning further surveillance of the epidemic, and developing pandemic prevention and containment strategies.


\section{Code and data availability}
All relevant data and script that generate the graphs are available in a dedicated github repository at \url{github.com/neherlab/CoV_seasonality}.

\section*{Acknowledgements}
We are grateful to all physicians, scientists, and public health authorities globally for sharing case numbers without delay.

\section*{Authors' contributions} \Robert{please add and modify}
JA conceived the idea of the study. RAN developed the model and simulations. JA and RD provided the data on seasonal CoVs. RAN, EBH and VD performed the modeling and simulation, and generated the graphs. All authors interpreted the data, drafted and commented on the manuscript, and contributed to the final version. 

\bibliography{bib}

\newpage

\appendix
\setcounter{figure}{0}
\renewcommand{\figurename}{Figure S}
\setcounter{table}{0}
\renewcommand{\tablename}{Table S}

\section*{Supplementary information}

\subsection{SIR model}

The SIR model introduced in equations \ref{eq:SIR} describe the fraction of susceptible $S(t)$ and infected individuals $I(t)$ in a population. 
The fraction of recovered individuals $R$ is $R=1 - S - I$. 
A simplified scheme of the model is presented in figure \ref{fig:schemeSIR} with constant transmissibility ($\epsilon=0$ in equation \ref{eq:transmissibility}) and without emigration/influx ($e=0$ and $i=0$).
The simplified system has a steady state
\begin{equation}
\begin{split}
S_0 &= \frac{\nu + b}{\beta} = \frac{1}{R_0} \\
I_0 & = \frac{b(R_0-1)}{\beta}
\end{split}
\label{eq:fixed_point}
\end{equation}
This steady state exists when the parameter $R_0 > 1$. For cases where $R_0 < 1$ the epidemic dies out as each infected person generate less than one subsequent infection. 

\begin{figure}[htb]
	\centering
	\includegraphics[width=0.9\columnwidth]{figures/SIR}
	\caption{Illustration of the SIR model used with its parameters.}\label{fig:schemeSIR}
\end{figure}


For our analysis of seasonal CoVs, it is instructive to study the stability of this steady state and show that the quantities $[S,I]$ tend to show damped oscillations around the fixed points in Eq.~\ref{eq:fixed_point} with a period
\begin{equation}
T = \frac{4\pi}{\sqrt{4b(\nu(R_0-1) - b) - R_0^2b^2}}
\label{eq:period}
\end{equation}
This behavior is intrinsic to the system and holds whenever transmissibility is constant.

SIR models can show resonance phenomena when the period is about one or two years \citep{dushoff_dynamical_2004,chen_regular_2017}.
Resonance is less likely when the period is much longer than the the period of annual forcing.
Fig.~\ref{fig:intrinsicPeriod} shows the intrinsic period for the parameter range considered. 
In the center of the graph the intrinsic period is around 3 years, suggesting that resonance is not strong.

In a fully susceptible population, the number of infected individuals initially grows as $I(t)\sim e^{(\beta-\nu-b)t} = e^{(R_0-1)t/(\nu+b)}$.
The doubling time is $\tau_2 = (\nu+b)\log(2)/(R_0-1)$.
Since $b\ll \nu$, we can safely neglect $b$ in the early phase of the pandemic, but it is important when interpreting endemic seasonal CoV data. 


\begin{figure}[htb]
	\centering
	\includegraphics[width=0.9\columnwidth]{figures/Period_phase_space}
	\caption{Length of the intrinsic period of the SIR model for $\beta$ and $\nu$ values in the range explored in this paper. The duration of immunity is assumed to be $10$ years. The dark blue region corresponds to $R_0<1$. }
	\label{fig:intrinsicPeriod}
\end{figure}


\subsection*{Migration between subpopulations}

The term $i$ and $e$ in equation \ref{eq:SIR} represent the influx and emigration in a population which we model as an exchange of infected individuals while ignoring the migration of susceptible or recovered individuals.
Such migration is operating in figures \ref{fig:nCoV_predictions}, \ref{fig:global_3_panel} and \ref{fig:endemic}. 
A proportion $e$ of the infected people are removed due to emigration while an influx $i$ is added.
The influx $i$ of each population is computed as the product of the total number of infected people in all population, the migration rate of the population in question, and its size. 
In that sense, the infected persons form a global ``reservoir'' from which each population takes its influx.

\subsection*{Effect of infection control measures}
The SIR model as formulated in Eq.~\ref{eq:SIR} accounts for seasonality but otherwise assumes the transmission dynamics is constant. 
In reality, individuals change their behavior when they are aware of a contagious disease.
This in particularly true for the current SARS-CoV-2 outbreak where 10s of millions of people are under quarantine measures and travel restrictions.
In order to account for such the behavorial changes, we add a factor $H$ to equation \ref{eq:SIR} that modulates transmission:
\begin{equation}
\frac{d}{dt} I =  -(\nu+b+e) I + (1-H)\beta S I + i
\end{equation}
where $H = c\cdot\frac{I^3}{(K^3+I^3)}$ with $c$ the containment value. The term $H$ is a Hill function of order 3 and inflection point $K$. 
We use this to model a decrease in the rate of new infection smoothly decrease by a factor $c$. 
We used $c=0.5$ and $K=0.03$, implying that once a disease is at a prevalence of 3\%, containment measures reduce transmission by 50\%.
The effect of prevention and control is difficult to assess at present and will certainly vary in time and space. 

% \subsection{Simulation checks}
% \begin{itemize}
% 	\item Equation for the system (dS/dt and dS/dI) are alright.
% 	\item Computation of peak ratio and peak finding algorithm checked. They work fine. We only see a slight variation when using the scipy.signal.find\_peaks dedicated function (probably due to the fact that we use the whole vector instead of 1 every 10 elements in this case).
% 	\item Computation of emigration / influx is consistent with a global reservoir of infected people from which each population takes its influx.
% \end{itemize}

\begin{figure*}[h]
	\centering
	\includegraphics[width=0.9\textwidth]{figures/scenario_subplot.pdf}
	\caption{Same as figure \ref{fig:nCov_predictions} - but with varying $\langle R_0\rangle$ from 1.3-3 and Migration from 0.1\% to 10\% per year. Higher migration rates result in earlier introductions and higher likelihood of a peak in early 2020. Similarly, higher $\langle R_0\rangle$ results in more rapid growth and a higher likelihood of an early peak.}
	\label{fig:scenarios_supp}
\end{figure*}

\begin{figure*}[h]
	\centering
	\includegraphics[width=0.9\columnwidth]{{figures/peak_ratio_0.3}.pdf}
	\includegraphics[width=0.9\columnwidth]{{figures/peak_ratio_0.7}.pdf}
	\caption{Same as figure \ref{fig:nCov_predictions} with $\epsilon=0.3$ and $\epsilon=0.7$.}
	\label{fig:peak_ratio_supp}
\end{figure*}

\begin{figure*}[h]
	\centering
	\includegraphics[width=\textwidth]{{figures/global_3_panel_1.4}.pdf}	\includegraphics[width=\textwidth]{{figures/global_3_panel_2.7}.pdf}
	\caption{Same as figure \ref{fig:global_3_panel} with average $\langle R_0\rangle=1.4$ (top) and 2.7 (bottom)}
	\label{fig:global_3_panel_supp}
\end{figure*}

\newpage

\begin{figure*}[h]
	\centering
	\includegraphics[width=0.8\columnwidth]{{figures/global_endemic_1.4}.pdf}
	\includegraphics[width=0.8\columnwidth]{{figures/global_endemic_2.7}.pdf}
	\caption{Same as figure \ref{fig:endemic} with $R_0=1.5$ and $R_0=3.0$}
\end{figure*}



% \begin{figure*}[h]
% 	\centering
% 	\includegraphics[width=0.9\columnwidth]{figures/pos_count_vs_time.pdf}
% 	\includegraphics[width=0.9\columnwidth]{figures/pos_frac_vs_age.pdf}
% 	\caption{Same as figure \ref{fig:seasonal_CoV} in counts instead of ratios or by age of sample instead of month of sampling. \Robert{very interesting figure with the peak at 30-40 years! but- we don't really comment or discuss this in the context of modelling so I'm a bit hesitant to put it in this manuscript}}
% \end{figure*}

\end{document}



%\begin{figure}
    %\centering
    %\includegraphics[width=\columnwidth]{figures/Graph_temp.pdf}
%    \caption{%Fraction of samples submitted for diagnostics being positive for either "common" corona virus correlated to average monthly temperature.  \robert{my calculation of fraction positives is not comparable over time as number of analyses changes from 4 to 3 when HKU1 is analysed together with OC43...}
%}
%\label{fig:temp_corr}
%\end{figure}


%\begin{figure}
%    \centering
    %\includegraphics[width=\columnwidth]{figures/Graph_season_box.pdf}
%    \caption{%Fraction of samples submitted for diagnostics being positive for either "common" corona virus.  \Robert{in response to  Richard's comment that only thing we know is there is a seasonal pattern, maybe this plot is better instead of avg temp? \robert{my calculation of fraction positive is not comparable over time as number of analyses changes from 4 to 3 when HKU1 is analysed together with OC43...}
%}
%    \label{fig:season_corr}
%\end{figure}